ci-scripts/compute_hash.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+import hashlib
+import binascii
+import argparse
+
+
+def calculate_binary_hash(path):
+    with open(path, "rb") as f:
+        binary = f.read()
+        hex_hash = hashlib.sha256(binary).hexdigest()
+        return "".join(['\\x' + hex_hash[i:i + 2] for i in range(0, len(hex_hash), 2)])
```

Comment:
Why are you returning the hash in this peculiar form?

Reply:
So then it could be easily copy-pasted in code.